torch/quantization/_numeric_suite_fx.py

```diff
@@ -137,3 +137,204 @@ def compare_weights_fx(float_dict, quantized_dict):
                 )
 
     return weight_dict
+
+def _get_logger_dict_helper(mod, target_dict, prefix=""):
```

Comment:
Can we import this function from eager mode numeric suite and use it here? Would be good to reuse the existing function if it can be shared by eager mode and fx.

Reply:
Done